### PR TITLE
tell me what to change tyman

### PR DESCRIPTION
**Pull request by TrashCan69420**
bad

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,31 @@
-# Cascade
-![discord badge](https://img.shields.io/badge/Discord-Join%20the%20discord-blue?style=for-the-badge&link=https://discord.gg/Dsue9NYRZs)
-<br>
-A bot framework for discord using discordeno
+<h1 align = "center"><img src="">imagie goes here once we get a better logo</h1>
+<h1 align="center">Cascade: Library for DiscordDeno</h1>
+
+<div align="center">
+
+ ![discord badge](https://img.shields.io/badge/Discord-Join%20the%20discord-blue?style=for-the-badge&link=https://discord.gg/Dsue9NYRZs)
```
Comment (TymanWasTaken) [inline README.md]: Get more badges because yes

File: README.md [hunk 2]
```diff
@@ -1,4 +1,31 @@
-# Cascade
-![discord badge](https://img.shields.io/badge/Discord-Join%20the%20discord-blue?style=for-the-badge&link=https://discord.gg/Dsue9NYRZs)
-<br>
-A bot framework for discord using discordeno
+<h1 align = "center"><img src="">imagie goes here once we get a better logo</h1>
+<h1 align="center">Cascade: Library for DiscordDeno</h1>
```
Comment (TymanWasTaken) [inline README.md]: Its Discordeno not DeicordDeno

**Review by TymanWasTaken — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (TrashCan69420) [reply README.md]: fuck

File: README.md (hunk 1 above)
Comment (TrashCan69420) [reply README.md]: which ones

**Comment by TrashCan69420** [comment]
@TymanWasTaken

**Comment by TrashCan69420** [comment]
GitHub is fucking weird

**Review by TymanWasTaken — APPROVED** [review]

**Pull request merged**